src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/VariantFmsDTOValidator.java

```diff
@@ -0,0 +1,325 @@
+		List<String> noteIdentities = new ArrayList<String>();
+		Boolean allNotesValid = true;
+		if (CollectionUtils.isNotEmpty(dto.getNotes())) {
+			for (int ix = 0; ix < dto.getNotes().size(); ix++) {
+				ObjectResponse<Note> noteResponse = variantNoteFmsDtoValidator.validateVariantNoteFmsDTO(dto.getNotes().get(ix));
+				if (noteResponse.hasErrors()) {
+					allNotesValid = false;
+					variantResponse.addErrorMessages("notes", ix, noteResponse.getErrorMessages());
+				} else {
+					String noteIdentity = NoteIdentityHelper.variantNoteFmsDtoIdentity(dto.getNotes().get(ix));
+					if (!noteIdentities.contains(noteIdentity)) {
+						noteIdentities.add(noteIdentity);
+						validatedNotes.add(noteDAO.persist(noteResponse.getEntity()));
+					}
+				}
+			}
+		}
+		if (!allNotesValid) {
+			variantResponse.convertMapToErrorMessages("notes");
+		}
+		if (CollectionUtils.isNotEmpty(validatedNotes) && allNotesValid) {
+			if (variant.getRelatedNotes() == null) {
+				variant.setRelatedNotes(new ArrayList<>());
+			}
+			variant.getRelatedNotes().addAll(validatedNotes);
+		}
+		
+		if (variantResponse.hasErrors()) {
+			variantResponse.convertErrorMessagesToMap();
+			throw new ObjectValidationException(dto, variantResponse.errorMessagesString());
+		}
+		
+		variant = variantDAO.persist(variant);
+		if (variant != null) {
+			idsAdded.add(variant.getId());
+		}
+
+		return variant;
+	}
+	
+	@Transactional
+	public void validateCuratedVariantGenomicLocationAssociation(VariantFmsDTO dto, List<Long> idsAdded, Variant variant) throws ValidationException {
+
+		CuratedVariantGenomicLocationAssociation association = new CuratedVariantGenomicLocationAssociation();	
+		ObjectResponse<CuratedVariantGenomicLocationAssociation> cvglaResponse = new ObjectResponse<CuratedVariantGenomicLocationAssociation>();
+		AssemblyComponent chromosome = null;
+		
+		if (dto.getStart() == null) {
+			cvglaResponse.addErrorMessage("start", ValidationConstants.REQUIRED_MESSAGE);
+		} 
+		
+		if (dto.getEnd() == null) {
+			cvglaResponse.addErrorMessage("end", ValidationConstants.REQUIRED_MESSAGE);
+		}
+		
+		if (StringUtils.isBlank(dto.getSequenceOfReferenceAccessionNumber())) {
+			cvglaResponse.addErrorMessage("sequenceOfReferenceAccessionNumber", ValidationConstants.REQUIRED_MESSAGE);
+		} else {
+			ChromosomeAccessionEnum cae = ChromosomeAccessionEnum.getChromosomeAccessionEnum(dto.getSequenceOfReferenceAccessionNumber());
+			if (cae != null) {
+				Map<String, Object> params = new HashMap<>();
+				params.put("name", cae.chromosomeName);
+				params.put("genomeAssembly.modEntityId", cae.assemblyIdentifier);
+				SearchResponse<AssemblyComponent> acResponse = assemblyComponentDAO.findByParams(params);
+				if (acResponse != null) {
+					chromosome = acResponse.getSingleResult();
+				}
+			}
+			if (chromosome == null) {
+				cvglaResponse.addErrorMessage("sequenceOfReferenceAccessionNumber", ValidationConstants.INVALID_MESSAGE + " (" +
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'+' should be on a new line.
<!-- __reviewdog__:ChA3MDUzMzlmMGE4OTI0MmUyEgpjaGVja3N0eWxl -->